publish.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+rm -rf ./www/
+rm -rf ./dist/
+npm run prod-electron
+cp ./package.json ./www/
+sed  -i'' '6,999d' ./www/package.json
```

Comment:
`6,$d`?

Reply:
@l2dy 

It was hard to explain. When you first running `electron-builder`, the first step of it is to read a config from your package.json in your current folder. If you did not specify argument `project`, it will search for the package.json with the config file in the only current folder.

After it successfully loads the config and it will start building it. Now in this project, I have specified another argument called `app` which will let the electron builder build the entire `www` folder. Notice that currently, it has already loaded the entire config while it also requires another package.json file in `www` folder. That package.json absolutely could not have a `build` object about the config. And also could not have a `post-install` script. If you have any dependencies it will also install and package those useless files.

Now for Kahla that the `www` folder is already the dist version of it and package.json is not here of course. So I have removed all those useless dependencies and build. Only copy those necessary properties to the `www` folder for electron-builder to read.

Reply:
I mean why would you use `999`? If you'd like to delete until the last line please use `$`.

Reply:
That was a great suggestion!